source/includes/stackpath/_workloads.md

```diff
@@ -117,6 +117,110 @@ Attributes | &nbsp;
+   "specs":"SP-1",
+   "persistenceStoragePath":"",
+   "persistenceStorageSize":1,
+   "deploymentName":"wi-root-ion",
+   "deploymentInstancePerPops":1,
+   "enableAutoScaling":false,
+   "deploymentPops":"YYZ"
+}
+
+// Request body example for a Container:
+{
+   "name":"w-user-pah",
+   "type":"CONTAINER",
+   "image":"nginx:latest",
+   "addImagePullCredentialsOption":true,
+   "containerUsername":"test.username",
+   "containerPassword":"test-password",
+   "containerServer":null,
+   "containerEmail":null,
+   "environmentVariableKey": null,
+   "environmentVariableValue": null,
+   "secretEnvironmentVariableKey":null,
+   "secretEnvironmentVariableValue":null,
+   "vpc":"Default",
+   "addAnyCastIpAddress":false,
+   "publicPorts":"80-200",
+   "protocol":"UDP",
+   "specs":"SP-3",
+   "persistenceStoragePath":null,
+   "persistenceStorageSize":1,
+   "deploymentName":"wi-root-pxa",
+   "deploymentInstancePerPops":1,
+   "deploymentPops":"FRA",
+   "enableAutoScaling":true,
+   "cpuUtilization":50,
+   "minInstancesPerPop":1,
+   "maxInstancesPerPop":2
+}
+```
+> The above commands return a JSON structured like this:
+```json
+{
+  "taskId": "7135ae25-8488-4bc5-a289-285c84a00a84",
+  "taskStatus": "PENDING"
+}
+```
+<code>POST /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/workloads</code>
+
+Create a new workload in a given [environment](#administration-environments).
+
+Required | &nbsp;
+------- | -----------
+ `name`<br/>*string* | The name of the workload.
+ `type`<br/>*string* | Specify whether a workload is a VM-based workload or container-based.
+ `image`<br/>*string* | Either the location of a Docker image to run as a container or the image to use for the virtual machine. If for a virtual machine, this is in the format of /[:]. If the image tag portion is omitted, 'default' is assumed which is the most recently created, ready, and non-deprecated image of that slug. A set of common images is present on the 'stackpath-edge' stack.
+ `vpc`<br/>*string* | The virtual private cloud option for now supports default vpc only.
+ `protocol`<br/>*string* | Supported protocols are: `TCP`, `UDP` and `TCP_UDP`.
+ `firstBootSshKey(s)`<br/>*string* | If creating a VM-based workload, SSH keys are required. Multiple SSH keys can be separated by commas.
+ `specs`<br/>*string* | Specification type for resources which are allocated to each instance in a workload. Supported specifications are `SP-1 (1 vCPU, 2 GB RAM)`,`SP-2 (2 vCPU, 4 GB RAM)`,`SP-3 (2 vCPU, 8GB RAM)`,`SP-4 (4 vCPU, 16 GB RAM)`,`SP-5 (8 vCPU, 32 GB RAM)`.
+ `deploymentName`<br/>*string* | The name of the deployment.
+ `deploymentPops`<br/>*string* | The point of presence of a deployment. In the format [A-Z][A-Z][A-Z].
+ `enableAutoScaling` <br/>*boolean* | Specify if you would like to enable autoscaling. If enabled, the following fields are required.
```

Comment:
```suggestion
 `enableAutoScaling` <br/>*boolean* | Specify if you would like to enable autoscaling. If enabled, then `cpuUtilization` , `minInstancesPerPop` and `maxInstancesPerPop` are required.
```

We might want to add more fields after `maxInstancesPerPop` and this way it won't confuse API consumers if those newly added fields after `enableAutoScaling` are required by `enableAutoScaling` or not.